### PR TITLE
HCAL Run3-related updates for HLT

### PR DESCRIPTION
**Pull request by Martin-Grunewald**
#### PR description:

HCAL Run3-related updates for HLT
Based on CMSSW_11_0_0_pre10
Importing the customisation as given in #28163 directly into ConfDB & the HLT menus.

#### PR validation:

TSG tests

#### if this PR is a backport please specify the original PR:

N/A

### CODE REVIEW

**Comment by Martin-Grunewald** [comment]
please test

**Comment by mariadalfonso** [comment]
@Martin-Grunewald 
I checked in details the  HLTrigger/Configuration/python/HLT_FULL_cff.py in this PR and the changes are in agreement with the customization suggested in my PR.

thanks

**Comment by Martin-Grunewald** [comment]
please test

**Comment by Martin-Grunewald** [comment]
+1

File: Configuration/HLT/python/autoCondHLT.py [hunk 1]
```diff
@@ -19,6 +19,7 @@
     'Fake1'        : ( ','.join( [ 'L1Menu_Collisions2015_25nsStage1_v5'                     , l1MenuRecord,connectionString, l1MenuLabel, "2015-10-26 12:00:00.000"] ), ),
     'Fake2'        : ( ','.join( [ 'L1Menu_Collisions2016_v9_m2_xml'                         ,l1tMenuRecord,connectionString,l1tMenuLabel, "2016-10-06 19:36:53.000"] ), ),
     'FULL'         : ( ','.join( [ 'L1Menu_Collisions2018_v2_1_0-d1_xml'                     ,l1tMenuRecord,connectionString,l1tMenuLabel, "2018-08-14 11:03:08.000"] ), ),
+    '2018'         : ( ','.join( [ 'L1Menu_Collisions2018_v2_1_0-d1_xml'                     ,l1tMenuRecord,connectionString,l1tMenuLabel, "2018-08-14 11:03:08.000"] ), ),
```
[the remark this answers is not in the document]
Comment (Martin-Grunewald) [reply Configuration/HLT/python/autoCondHLT.py]: 2018 is a clone of GRun BEFORE the introduction of the Hcal Run3 changes in GRun and the other development menus, corresponding to 2018 Hcal. The plan is to have it for TSG studies in 11_0 and remove it in 11_1.

[the remark this answers is not in the document]
Comment (fabiocos) [reply Configuration/HLT/python/autoCondHLT.py]: @Martin-Grunewald to keep GRun free for further changes I guess

[the remark this answers is not in the document]
Comment (Martin-Grunewald) [reply Configuration/HLT/python/autoCondHLT.py]: Indeed: GRun is for development toward Run-3.

**Comment by fabiocos** [comment]
+1

**Pull request merged**